SU-SWS/stanford_profile
# D8CORE-1315: splash fix

# READY FOR REVIEW

# Summary
fix the missing splash styles

# Needed By (Date)
- before sprint end

# Urgency
- needed for the splash styles to work

# Steps to Test

1. Check out this branch
1. Clear Cache
2. In the editor add "Splash Font" to text
3. Save.
4. Verify that the Splash text is large and bold.

![Screen Shot 2020-02-19 at 4 36 58 PM](https://user-images.githubusercontent.com/284440/74889572-581f3100-5336-11ea-9774-9053f257a183.png)


# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1315

- Anyone who should be notified? (`@mention` them here)
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
